core/test/fixtures/lantern-baseline-computed-values.json

```diff
@@ -1,81 +1,81 @@
 {
   "sites": [
-    {"url": "http://m.iciba.com", "roughEstimateOfFCP": 1743, "optimisticFCP": 1743, "pessimisticFCP": 1743, "roughEstimateOfFMP": 1743, "optimisticFMP": 1743, "pessimisticFMP": 1743, "roughEstimateOfTTI": 7517, "optimisticTTI": 3337, "pessimisticTTI": 11697, "roughEstimateOfSI": 9842, "optimisticSI": 4764, "pessimisticSI": 5264, "roughEstimateOfLCP": 9781, "optimisticLCP": 9341, "pessimisticLCP": 10222},
```

Comment:
I did this PR because I noticed a delta in this file when running `node core/scripts/lantern/update-baseline-lantern-values.js`. It was simply the new TTFB metric being added (nothing else changed, as expected). The other metrics do not show up yet because the necessary data in the trace is missing.

Reply:
Hmm why is TTFB added but not LCP load start/end? I would imagine that none of the sites had a baseline TTFB before now.

Reply:
There's likely no DOMNodeId in these old traces, so LCPImageRecord will fail to determine load start/end.

ttfb doesn't rely on any new trace stuff

Reply:
ah, they don't even have `LargestImagePaint::Candidate` events :P